src/playground-project.ts

```diff
@@ -712,6 +712,59 @@ const typeEnumToMimeType = (type?: string) => {
       return 'text/html; charset=utf-8';
     case 'css':
       return 'text/css; charset=utf-8';
+    // taken from MDN's common MIME types
+    // https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types
+    case 'svg':
+      return 'image/svg+xml';
+    case 'png':
+      return 'image/png';
+    case 'jpeg':
+    case 'jpg':
+      return 'image/jpeg';
+    case 'bmp':
+      return 'image/bmp';
+    case 'gif':
+      return 'image/gif';
+    case 'ico':
+      return 'image/vnd.microsoft.icon';
+    case 'tif':
+    case 'tiff':
+      return 'image/tiff';
+    case 'webp':
+      return 'image/webp';
+    case 'avi':
+      return 'video/x-msvideo';
+    case 'mp4':
+      return 'video/mp4';
+    case 'mpeg':
+      return 'video/mpeg';
+    case 'ogv':
+      return 'video/ogg';
+    case 'ts':
```

Comment:
We already have to map `ts` for typescript in a case above. We'll reach here, but let's not even include it (or maybe comment it out with a comment explaining that).